css/imac-section.css

```diff
@@ -0,0 +1,81 @@
+.imac-section {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+
+	padding: 4rem 0;
+
+	color: var(--black-color);
+}
+.imac-section .wrapper {
```

Comment:
Jeśli potrzebujesz w BEM uzależnić dany element od czegoś (nadać mu szczególne cechy to używać modyfikatora) czyli np. `.imac-section--wrapper`

Reply:
Nie rozumiem do końca jak ma to zadziałać. W moim przypadku kolor sekcji nadaję na tagu section, w momencie w którym dodam klasę wrapper w HTML czyli 

<section class="imac-section imac-section--wrapper">
<div class="wrapper"></div>
</section>

Div z sekcji będzie niepotrzebny, ale wówczas cała sekcja będzie mieć zadeklarowane właściwości. Takie jak 
width: 80%;

Wtedy też (może nie w tym przypadku, ale w przypadku innych sekcji) mój kolor backgroundu również będzie ograniczony do szerokości wrappera. Chyba, że w moim rozumowaniu czegoś brakuje